Xamarin.Forms.Platform.iOS/Renderers/LabelRenderer.cs

```diff
@@ -606,7 +615,39 @@ void UpdatePadding()
 #endif
 		}
 
+		void UpdateAutoFitMode()
+		{
 #if __MOBILE__
+
+			switch (Element.AutoFitText)
+			{
+				case AutoFitTextMode.FitToWidth:
+					Control.AdjustsFontSizeToFitWidth = true;
+
+					var uiFont = Element.ToUIFont();
```

Comment:
Are you sure this will retain custom fonts and other specifics that might be set at that time?

Reply:
Tried a couple of different things and seems to work. Also with icon fonts, although they do not seem to take up the full width?

Reply:
> Are you sure this will retain custom fonts and other specifics that might be set at that time?

Yeah, `Element.ToUIFont();` is a current implementation to set fonts from Xamarin.Forms.Label to Xamarin.IOS.UILabel.. so I've expected that works 😁

>  Also with icon fonts, although they do not seem to take up the full width?

Hmm, I don tried with icon fonts, that's a great test to write

Reply:
Icons and emojis seems like OK !🚀